std_msgs/Int8 house_ID
std_msgs/Int8 num_of_pizza